.classpath

```diff
@@ -4,7 +4,29 @@
 	<classpathentry kind="src" path="src/test/java"/>
 	<classpathentry kind="src" path="build/gen-java-src"/>
 	<classpathentry kind="con" path="com.google.gwt.eclipse.core.GWT_CONTAINER"/>
-	<classpathentry exported="true" kind="con" path="org.springsource.ide.eclipse.gradle.classpathcontainer"/>
+	<classpathentry kind="con" path="org.springsource.ide.eclipse.gradle.classpathcontainer" exported="true"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry kind="output" path="src/main/webapp/WEB-INF/classes"/>
+	<classpathentry kind="output" path="war/WEB-INF/classes"/>
+	<classpathentry sourcepath="/Users/mpayne/.gradle/caches/modules-2/files-2.1/org.tessell/tessell-user/2.18.7/c6dde38c8186a7980ebe38d804e56bd6e1fe1d54/tessell-user-2.18.7-sources.jar" kind="lib" path="/Users/mpayne/.gradle/caches/modules-2/files-2.1/org.tessell/tessell-user/2.18.7/6be59c76c5692ffaf71b9ebef25860df65b42548/tessell-user-2.18.7.jar"/>
```

Comment:
Should remove these lines, since they have /Users/mpayne in them, and then also in theory they shouldn't be needed due to the Gradle IDE "classpathcontainer" line on line 7.